src/cli/main.py

```diff
@@ -0,0 +1,291 @@
+    endpoint = _get_endpoint(f"jobTemplates/{name}", ctx.obj["resource_group"], ctx.obj["subscription_id"])
+
+    session = ctx.obj["session"]
+
+    resp = session.get(endpoint)
+    _output_result(resp)
+
+
+@cli.command()
+@click.pass_context
+@click.option("--name", "-n", help="Job template name.", required=True)
+@click.option("--vm-size", "-s", help="VM size.")
+@click.option("--priority", "-p", help="A comma separated list of LISA test priority to run.")
+@click.option("--case-name", "-c", help="A comma separated list of LISA test case to run.")
+@click.option(
+    "--location",
+    "-l",
+    default="westus3",
+    help="Job template location, it's not required to be changed.",
+)
+@click.option(
+    "--region",
+    "-r",
+    multiple=True,
+    default=["westeurope"],
+    help="Regions where the test resources will be provisioned.",
+)
+def create_template(
+    ctx: click.Context,
+    name: str,
+    vm_size: str,
+    priority: str,
+    case_name: str,
+    location: str,
+    region: list[str],
+) -> None:
+    """
+    Create a new test job template.
+
+    If --priority and --case-name are not set, only p0 tests will be run (smoke tests).
+    """
+    endpoint = _get_endpoint(f"jobTemplates/{name}", ctx.obj["resource_group"], ctx.obj["subscription_id"])
+
+    session = ctx.obj["session"]
+
+    template = _create_template(vm_size, priority, case_name, location, region)
+
+    payload = {"location": location, "name": name, "properties": template}
+    resp = session.put(endpoint, json=payload)
+    _output_result(resp)
+
+
+@cli.command()
+@click.pass_context
+def list_jobs(ctx: click.Context) -> None:
+    """
+    List all the test jobs.
+    """
+    endpoint = _get_endpoint("jobs", ctx.obj["resource_group"], ctx.obj["subscription_id"])
+
+    session = ctx.obj["session"]
+
+    resp = session.get(endpoint)
+    _output_result(resp)
+
+
+@cli.command()
+@click.pass_context
+@click.option("--name", "-n", required=True)
+def get_job(ctx: click.Context, name: str) -> None:
+    """
+    Get a test job. Can be used to get the current status of the job.
+    """
+    endpoint = _get_endpoint(f"jobs/{name}", ctx.obj["resource_group"], ctx.obj["subscription_id"])
+
+    session = ctx.obj["session"]
+
+    resp = session.get(endpoint)
+    _output_result(resp)
+
+
+@cli.command()
+@click.pass_context
+@click.option("--name", "-n", required=True, help="Job name.")
+@click.option(
+    "--marketplace-image-urn",
+    "-u",
+    help='URN of the image ("az vm image list -p Canonical --all").',
+)
+@click.option("--vhd-sas-url", "-v", type=str, help="SAS URL of a VHD to test.")
+@click.option("--architecture", "-a", type=click.Choice(["x64", "arm64"]), help="Architecture of the image.")
+@click.option("--vm-generation", "-g", default="2", type=click.Choice(["1", "2"]), help="Hyper-V generation.")
+@click.option(
+    "--region", "-r", default=["westeurope"], type=str, multiple=True, help="Provisioning region for test resources."
```

Comment:
As above, should this parameter also take a comma-separated list in the interest of consistency with `--priorities` and `--case-names`?

Reply:
I will do the opposite as suggested by Aleksa and make the two others use `multiple`.